### PR TITLE
Mohammad

### PR DESCRIPTION
**Pull request by markazie**

### CODE REVIEW

**Comment by Alisina92** [comment]
Hi Mohammad try to give meaningful name to your project like Mohammad - week1 js excercise.

File: week-1/G-strings-methods/exercise2.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 var name = " Daniel  ";
 
-console.log(message);
+console.log('My name is ' + name.trim() +  ' and my name is 6 characters long');
```
Comment (martynamaron) [inline week-1/G-strings-methods/exercise2.js]: Not sure this will print the desired result? :D

File: week-1/E-strings/exercise.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+message = "Hi";
```
Comment (martynamaron) [inline week-1/E-strings/exercise.js]: Even though this compiles and runs ok, it's not the right way of initialising variables in JS - could you please fix that?

File: week-1/F-strings-concatenation/exercise.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+message = "Hello, my name is";
```
Comment (martynamaron) [inline week-1/F-strings-concatenation/exercise.js]: Same as above.

File: week-1/G-strings-methods/exercise.js [hunk 4]
```diff
@@ -1,3 +1,3 @@
 // Start by creating a variable `message`
-
-console.log(message);
+name = 'Jack';
```
Comment (martynamaron) [inline week-1/G-strings-methods/exercise.js]: Same as above.

**Comment by martynamaron** [comment]
Hey Mohammad 👋 
Great that you've opened this PR! However, I see you've only got the class exercises in here - could you add on your homework to it please? Do let us know if you're having trouble though!
Happy coding :)

File: week-1-practise/level-1/8-function-output-1.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? get a random number between 0 and 1; return random number times by 10.
+// - How many parameters does the function take? 2
```
Comment (LucyMac) [inline week-1-practise/level-1/8-function-output-1.js]: Parameters are values that are given to the function _inside_ the `()`.  Is there anything inside the parentheses of `getNumber()`?

File: week-1-practise/level-1/9-function-output-2.js [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 // Answer these questions:
-// - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// - What does this program do? Run it and see, or Google some of the methods used ot understand them. concat 2 words
+// - How many parameters does the function take? 2
```
Comment (LucyMac) [inline week-1-practise/level-1/9-function-output-2.js]: Taking my comment above in mind, would you change your answer here? :)

File: week-1-practise/level-1/9-function-output-2.js [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
+// - What is the function name? s
+// - Where is the function called? with what parameters? line 5 - result variable 
+// - What does the function return? one concated word
+// - How can this function be improved? by choose proper name
```
Comment (LucyMac) [inline week-1-practise/level-1/9-function-output-2.js]: Good

File: week-1-practise/level-3/1-piping.js [hunk 8]
```diff
@@ -16,16 +16,18 @@ var startingValue = 5;
 /* --------------------------- */
 
 function add(a, b) {
+  b = 10;
   return a + b;
```
Comment (LucyMac) [inline week-1-practise/level-3/1-piping.js]: Not quite. You should not be reassigning a new value to an argument. Instead, you should provide that value _when you call the function_.
Same problem for `multiply` below. 
Can you think of how to do this?

File: week-1-practise/level-3/1-piping.js [hunk 9]
```diff
@@ -16,16 +16,18 @@ var startingValue = 5;
...
 }
 
 function multiply(a, b) {
+  b = 2;
```
Comment (LucyMac) [inline week-1-practise/level-3/1-piping.js]: See my comment above. 
You have changed the definition of these functions, but you are not calling them, which means they are not running.

Comment (LucyMac) [reply week-1-practise/level-3/1-piping.js]: Have a look here and see the difference between *defining* what a function should do (this never changes), and *calling* a function (= using it to do something).

File: week-1/G-strings-methods/exercise.js (hunk 4 above)
Comment (LucyMac) [reply week-1/G-strings-methods/exercise.js]: Mohammad, can you fix all your variables as requested by Martyna?

File: week-1/I-floats/exercise.js [hunk 10]
```diff
@@ -1,2 +1,8 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+
+PercentageStudents =  Math.round((numberOfStudents*100)/total);
+PercentageMentors = Math.round((numberOfMentors*100)/total);
+
```
Comment (LucyMac) [inline week-1/I-floats/exercise.js]: Again, you must declare your variables correctly.

**Pull request closed without merging**